src/main/java/org/pmiops/workbench/model/Cohort.java

```diff
@@ -0,0 +1,25 @@
+package org.pmiops.workbench.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Cohort {
```

Comment:
What do you think of using protos for models? If database interaction will deal with it, it gets us builders and (de)serialization. It looks like proto has a JSON formatter in Java (https://github.com/google/protobuf/blob/master/java/util/src/main/java/com/google/protobuf/util/JsonFormat.java#L97) if we want to keep speaking JSON externally.